.rubocop_todo.yml

```diff
@@ -1,44 +1,387 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2016-10-28 16:34:29 +0300 using RuboCop version 0.44.1.
-
+# on 2018-03-12 10:41:09 +0000 using RuboCop version 0.53.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-Metrics/AbcSize:
-  Max: 16
+# Offense count: 1
+# Cop supports --auto-correct.
+# Configuration parameters: Include, TreatCommentsAsGroupSeparators.
+# Include: **/*.gemfile, **/Gemfile, **/gems.rb
+Bundler/OrderedGems:
   Exclude:
-    - 'lib/ossert/fetch/github.rb'
+    - 'Gemfile'
 
-Metrics/BlockLength:
+# Offense count: 9
+# Cop supports --auto-correct.
+# Configuration parameters: Include, TreatCommentsAsGroupSeparators.
+# Include: **/*.gemspec
+Gemspec/OrderedDependencies:
   Exclude:
     - 'ossert.gemspec'
-    - 'lib/ossert/tasks/ossert.rake'
-    - 'lib/ossert/tasks/database.rake'
-    - 'spec/ossert_spec.rb'
-    - 'spec/spec_helper.rb'
 
-Metrics/ClassLength:
+# Offense count: 1
+# Cop supports --auto-correct.
+Layout/CommentIndentation:
+  Exclude:
+    - 'lib/ossert/repositories.rb'
```

Comment:
Hmm. Why do I see there much more _Exclude:_ then before?
Also, most of them support _auto-correct_ option.
Maybe, it's worth to run `rubocop -a` and commit (and review) those changes.